fuseutil/dirent.go

```diff
@@ -103,3 +117,136 @@ func WriteDirent(buf []byte, d Dirent) (n int) {
 
 	return n
 }
+
+// Write the supplied directory entry with attributes into the given buffer in the format
+// expected in fuseops.ReadDirPlusOp.Dst returning the number of bytes written.
+// Return zero if the entry would not fit.
+func WriteDirentPlus(buf []byte, d DirentPlus) (n int) {
+	type fuse_attr struct {
```

Comment:
This struct seems to duplicate `fusekernel.Attr`? Could we use `fusekernel.Attr` instead?)